### PR TITLE
@ #1 | should give direct edit rights to variables

### PR DESCRIPTION
**Pull request by datphan**

### CODE REVIEW

**Comment by hoatle** [comment]
@datphan what is the use case for this?

**Comment by datphan** [comment]
it will use in iorad-dev usecase that I will introduce it later,

some time we don't want to do this `KEY=value vagrant up` just to have the KEY env, in some case we just want it to permanently exists

config_override.yaml:
```
variables:
    gitlab_username: ...

```

**Comment by hoatle** [comment]
Users can override with default value for the case and they can use env var if they want.

**Comment by datphan** [comment]
isn't this
```
variables:
    gitlab_username: abc

username: "%{gitlab_username}"
```
better than when we dont want to export `GITLAB_USERNAME`?
```
variables:
    gitlab_username: ${GITLAB_USERNAME}

username: "%{gitlab_username}"
```

**Comment by hoatle** [comment]
variables:
    gitlab_username: ${GITLAB_USERNAME:-abc}

username: "%{gitlab_username}"

That’s how the abc value should be set

**Comment by hoatle** [comment]
For your case, if env var is not expected, just use:

username: abc

**Comment by datphan** [comment]
`gitlab_username: ${GITLAB_USERNAME:-abc}` is just longer and in case
```
gitlab_username: GITLAB_USERNAME
```
if `GITLAB_USERNAME` is not present, then the app will raise error if the config have `%{gitlab_username}`
we should cover this case

**Comment by hoatle** [comment]
For the shortest, use:

username: abc

I don’t see the reason to use “variables” when you want to set value directly, then just set it with the shortest form.

**Comment by datphan** [comment]
it has other use:
```
variables:
  gitlab_username: GITLAB_USERNAME

username: %{gitlab_username}
path: /path/to/%{gitlab_username}
...

```

**Comment by hoatle** [comment]
Ok, I see the use case, then we’ll support this.

Make sure to update code comment for this feature so that other devs can use, for docs writers to refer

**Review by hoatle — CHANGES_REQUESTED** [review]
https://github.com/teracyhq-incubator/teracy-dev-core/blob/develop/config.yaml#L1-L5

make sure to update code comment above for this feature

File: config.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # format:
 # key: ${ENV_VAR}
 # key: ${ENV_VAR-:default}
+# key: value
```
Comment (datphan) [inline config.yaml:5]: updated, pls check it

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**